File: Entity/Pedido.cs
using System;
using System.Collections.Generic;

namespace Entity
{
    public class Pedido
    {
        public decimal CodigoPedido{get;set;}
        public string FechaPedido{get;set;}
        public string HoraPedido{get;set;}
        public decimal TotalPedido{get;set;}
        public string IdCliente{get;set;}
        public List<Detalle> Detalles {get;set;}
        public string Estado {get;set;}

        public void CalcularTotal(){

            foreach (var item in Detalles)
            {
              TotalPedido = TotalPedido + item.TotalDetalle;   
            }
        }

        public void EstadoInicial(){

            Estado = "Recibido";
        }
    }
}

File: floridaimportdotnet/ClientApp/src/app/florida/models/pedido.ts
import { Detalle } from "./detalle";

export class Pedido {

    codigoPedido: number;
    fechaPedido: string;
    horaPedido: string;
    totalPedido: number;
    idCliente: string;
    detalles = new Array<Detalle>();
    estado: string;
}
